ssl/methods.c

```diff
@@ -228,17 +217,17 @@ const SSL_METHOD *TLSv1_client_method(void)
 # ifndef OPENSSL_NO_SSL3_METHOD
 const SSL_METHOD *SSLv3_method(void)
 {
-    return sslv3_method();
+    return NULL;
 }
 
 const SSL_METHOD *SSLv3_server_method(void)
 {
-    return sslv3_server_method();
+    return NULL;
 }
 
 const SSL_METHOD *SSLv3_client_method(void)
```

Comment:
These `SSLv3*` functions are now dead code. I assume you are retaining them for ABI compatibility? But you have also removed the enable-ssl3 option from Configure. So it is no longer possible to build a version of OpenSSL where these functions are available.

If ABI compatibility is a goal, then we cannot remove `enable-ssl3` from Configure. If it is not a goal and we retain the deletion of `enable-ssl3` from Configure then we should simply remove these functions.